youtube_dl/extractor/ceskatelevize.py

```diff
@@ -15,11 +19,13 @@
     unescapeHTML,
     urlencode_postdata,
     USER_AGENTS,
+    RegexNotFoundError,
+    compat_str,
 )
 
 
 class CeskaTelevizeIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?ceskatelevize\.cz/ivysilani/(?:[^/?#&]+/)*(?P<id>[^/#?]+)'
+    _VALID_URL = r'https?://(?:(?:www\.)?ceskatelevize\.cz/ivysilani/|mshokej\.ceskatelevize\.cz/)(?:[^/?#&]+/)*(?P<id>[^/#?]+)'
```

Comment:
Don't. Must be a separate article extractor.

Reply:
Separate for different domain names?